server/.envrc.example

```diff
@@ -25,7 +25,7 @@ export DOMAIN_OVERRIDE=localhost:8000
 # export ADMIN_EMAIL_EMAIL_TEST=""
 # export ADMIN_UIDS=[]
 
-# These may be the deprecated settings for submitting web requests to the math worker
+# These are the shared credentials for the math worker to use when polling
 export WEBSERVER_USERNAME=ws-user
 export WEBSERVER_PASS=ws-pass
```

Comment:
I don't think these are needed any more and can be deleted. This was for when the server directly made web requests to an the darwin server, which ran from the math codebase (but wasn't the same as the polling process). Now, export requests are handled through messasges sent through the postgres db.